src/formGenerator.vue

```diff
@@ -301,10 +310,16 @@ div.vue-form-generator(v-if='schema != null')
 			// Validating the model properties
 			validate() {
 				this.clearValidationErrors();
+				this.touched.splice(0);
 
 				this.$children.forEach((child) => {
 					if (isFunction(child.validate))
 					{
+						this.touched.push({
```

Comment:
Yes, I understand, as I said, this was my first idea. But how do you access `touched` from `formGenerator` ?
I was going to add an index to the v-for loop (`(field, index) in fields`), but I noticed that groups will break this (I just noticed groups are not working how I assumed they would and this is another problem).
Anyway, changing an internal `touched` property is not the problem, accessing it from the parent (`formGenerator`) is the difficulty. How to match the children with the parent ?